obsreport/obsreporttest/obsreporttest.go

```diff
@@ -41,19 +46,51 @@ var (
 	processorTag, _ = tag.NewKey("processor")
 )
 
+type TestTelemetrySettings struct {
```

Comment:
I have second thoughts about this between TestTelemetrySettings and TestTelemetry